types/lib/utils/async.d.ts

```diff
@@ -43,20 +43,6 @@ declare namespace Polymer {
       function run(fn: Function, delay?: number): number;
 
 
-      /**
-       * Enqueues a function called in the next task.
-       *
-       * @returns Handle used for canceling task
-       */
-      function run(fn: Function, delay?: number): number;
```

Comment:
Oh, I guess those `@function` annotations were important!

Reply:
these were duplicates, i think we still have them elsewhere in the file. `@function` caused the generator to output `run` twice...